MacaoProject/Macao/Card.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Macao
+{
+    public class Card
+    {
+        public string type { get; set; }
```

Comment:
read about naming conventions for properties https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names